File: Math/Factorial Trailing Zeroes/TrailingZeros.java
class Solution {
    public int trailingZeroes(int n) {
        int sum = 0;

        if (n < 5) {
            return 0;
        }
        while (n >= 5) {
            sum = sum + n / 5;
            n = n / 5;
        }
        return sum;
    }

}

File: Math/Factorial Trailing Zeroes/trailing_zeros.py
class Solution:
    def trailingZeroes(self, n: int) -> int:
        if n < 5:
            return 0

        count = 0
        for i in range(1, n + 1):
            j = i
            while j % 5 == 0:
                count += 1
                j //= 5
        return count
